app/assets/javascripts/nfg_ui/datetimepicker_configuration.js

```diff
@@ -79,4 +80,4 @@ function initNfgUiDateTimePicker() {
 if (NfgUi.turbolinks)
   $(document).on('turbolinks:load', initNfgUiDateTimePicker);
 else
-  $(document).ready(initNfgUiDateTimePicker);
\ No newline at end of file
+  $(document).ready(initNfgUiDateTimePicker);
```

Comment:
@farhajawed the datepicker area will need to be updated or at least reviewed to ensure that the documentation matches the new functionality. cc: @kylejmorin since he was just working on this.